sdk/core/core-http/src/policies/exponentialRetryPolicy.ts

```diff
@@ -139,6 +140,15 @@ async function retry(
 ): Promise<HttpOperationResponse> {
   function shouldPolicyRetry(responseParam?: HttpOperationResponse): boolean {
     const statusCode = responseParam?.status;
+    if (
+      statusCode &&
+      statusCode === 503 &&
```

Comment:
pretty sure `statusCode && statusCode === 503` is functionally the same as `statusCode === 503`